Documentation/subcommands/config.md

```diff
@@ -0,0 +1,130 @@
+# rkt config
+
+This command prints the rkt configuration per stage in JSON format.
+
+## Structure
+
+The general structure is a simple hierarchy consisting of the following top-level elements:
+
+```
+{
+	"stage0": [...],
+	"stage1": [...]
+}
+```
+
+The entries "stage0" and "stage1" refer to stage-specific configuration. The generated output are valid configuration entries as specified in the configuration documentation.
+
+The "stage0" entry contains entries of rktKind "auth", "dockerAuth", and "paths". Note that the `config` subcommand will output separate entries per "auth" domain and separate entries per "dockerAuth" registry. While it is possible to specify an array of strings in the input configuration rkt internally merges configuration state from different directories potentially creating multiple entries.
```

Comment:
Also, about the multiple `auth` entries - that's an implementation detail. We may change the code later to be smarter and merge the blocks with the same credentials. So I would keep quiet about it to avoid giving an impression that "domains" key is always a table with only one element.